hermes-frontend/src/test/groovy/pl/allegro/tech/hermes/frontend/publishing/preview/MessagePreviewLogTest.groovy

```diff
@@ -89,4 +109,10 @@ class MessagePreviewLogTest extends Specification {
         executorService.shutdown()
     }
 
+    static ObjectMapper createObjectMapper() {
```

Comment:
There are no usages of this objectMapper, can you remove it?

Reply:
ok, removed